Index: src/main/java/it/csi/siac/siacbilser/frontend/webservice/msg/InserisceDettaglioResiduoVariazioneBilancio.java
===================================================================
--- src/main/java/it/csi/siac/siacbilser/frontend/webservice/msg/InserisceDettaglioResiduoVariazioneBilancio.java	(revision 0)
+++ src/main/java/it/csi/siac/siacbilser/frontend/webservice/msg/InserisceDettaglioResiduoVariazioneBilancio.java	(revision 0)
@@ -0,0 +1,41 @@
+package it.csi.siac.siacbilser.frontend.webservice.msg;
+
+import it.csi.siac.siacbilser.frontend.webservice.BILSvcDictionary;
+import it.csi.siac.siacbilser.model.VariazioneImportoCapitolo;
+import it.csi.siac.siaccorser.model.ServiceRequest;
+
+import javax.xml.bind.annotation.XmlType;
+
+/**
+ * Classe di request per l'inserimento dell'anagrafica della variazione di bilancio
+ * @author Pro Logic
+ *
+ */
+@XmlType(namespace = BILSvcDictionary.NAMESPACE)
+public class InserisceDettaglioResiduoVariazioneBilancio extends ServiceRequest {
+	
+	private VariazioneImportoCapitolo variazioneImportoCapitolo;
+
+	/**
+	 * Gets the variazione importo capitolo.
+	 *
+	 * @return the variazione importo capitolo
+	 */
+	public VariazioneImportoCapitolo getVariazioneImportoCapitolo() {
+		return variazioneImportoCapitolo;
+	}
+
+	/**
+	 * Sets the variazione importo capitolo.
+	 *
+	 * @param variazioneImportoCapitolo the new variazione importo capitolo
+	 */
+	public void setVariazioneImportoCapitolo(VariazioneImportoCapitolo variazioneImportoCapitolo) {
+		this.variazioneImportoCapitolo = variazioneImportoCapitolo;
+	}
+	
+	
+	
+	
+
+}
Index: src/main/java/it/csi/siac/siacbilser/frontend/webservice/msg/InserisceDettaglioResiduoVariazioneBilancioResponse.java
===================================================================
--- src/main/java/it/csi/siac/siacbilser/frontend/webservice/msg/InserisceDettaglioResiduoVariazioneBilancioResponse.java	(revision 0)
+++ src/main/java/it/csi/siac/siacbilser/frontend/webservice/msg/InserisceDettaglioResiduoVariazioneBilancioResponse.java	(revision 0)
@@ -0,0 +1,14 @@
+package it.csi.siac.siacbilser.frontend.webservice.msg;
+
+import javax.xml.bind.annotation.XmlType;
+
+import it.csi.siac.siacbilser.frontend.webservice.BILSvcDictionary;
+import it.csi.siac.siacbilser.model.Cronoprogramma;
+import it.csi.siac.siaccorser.model.ServiceResponse;
+
+@XmlType(namespace = BILSvcDictionary.NAMESPACE)
+public class InserisceDettaglioResiduoVariazioneBilancioResponse extends ServiceResponse {
+
+		
+	
+}
